resources/packaging/opensuse/setup.sh.in

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+cp $ungoog{unpacked_src_dir}/ungoogled_packaging/patches/*/*.patch ~/rpm/SOURCES/
+cp $ungoog{unpacked_src_dir}/ungoogled_packaging/patches/*/*/*.patch ~/rpm/SOURCES/
+cp $ungoog{unpacked_src_dir}/ungoogled_packaging/master_preferences ~/rpm/SOURCES/
+cp $ungoog{unpacked_src_dir}/ungoogled_packaging/chromium-browser.sh ~/rpm/SOURCES/
+cp $ungoog{unpacked_src_dir}/ungoogled_packaging/chromium-browser.desktop ~/rpm/SOURCES/
+cp $ungoog{unpacked_src_dir}/ungoogled_packaging/chromium-browser.xml ~/rpm/SOURCES/
+cp $ungoog{unpacked_src_dir}/ungoogled_packaging/chromium.default ~/rpm/SOURCES/
+cp $ungoog{unpacked_src_dir}/ungoogled_packaging/chromium-browser.appdata.xml ~/rpm/SOURCES/
+cp $ungoog{unpacked_src_dir}/ungoogled_packaging/chromium-icons.tar.bz2 ~/rpm/SOURCES/
```

Comment:
Maybe something like this would be better:

1. Create a directory called `resources/packaging/opensuse/sources_template`
2. During the buildkit `genpkg` step, copy `sources_template/` to `ungoogled_packaging/SOURCES` using `shutil.copytree()`, and then process `.in` files using `process_templates()` from `buildkit.packaging._common`. Then, just `cp -r $(dirname $(readlink -f $0))/SOURCES/* ~/rpm/SOURCES/`
3. Assuming the contents of `chromium-icons.tar.bz2` lives in `resources/packaging/opensuse/chromium-icons_contents/`, copy it using `shutil.copytree()` to `ungoogled_packaging/chromium-icons_contents/`. Then something like the following code can generate the `.tar.bz2`:

    ```shell
    pushd $(dirname $(readlink -f $0))/chromium-icons_contents
    tar cjf ~/rpm/SOURCES/chromium-icons.tar.bz2 *
    popd
    ```